src/resources/dicts/it.edn

```diff
@@ -507,7 +507,7 @@
  :header/more "Altro"
  :header/search "Cerca"
  :header/toggle-left-sidebar "Attiva/disattiva il pannello laterale sinistro"
- :help/awesome-logseq "Awesome Logseq"
+ :help/awesome-logseq "Logseq fantastico"
```

Comment:
I prefer not to translate this. "Awesome xxx" is an idiom.

Reply:
As you prefer, but the literal translation is that.
Regards,
A.

Reply:
I agree with @andelf, the "Awesome" lists are called that way, no matter the language.